lib/file-info-view.coffee

```diff
@@ -10,10 +12,29 @@ class FileInfoView extends HTMLElement
     @bufferModified.classList.add('buffer-modified')
     @appendChild(@bufferModified)
 
+    @newTooltip()
+
     @activeItemSubscription = atom.workspace.onDidChangeActivePaneItem =>
       @subscribeToActiveItem()
     @subscribeToActiveItem()
 
+    clickHandler = ->
+      atom.clipboard.write(@getActiveItem()?.getPath?() or 'untitled')
+      setTimeout =>
+        @newTooltip()
+      , 2000
+
+    @addEventListener('click', clickHandler)
```

Comment:
I was considering that briefly, and it may be the better way. 
But then I thought, while we copy only the path, we click on the whole FileInfoView, which has exactly one purpose: display the path. 

So clicking anywhere in that tiny box that is the FileInfoView probably means you want to use the copy feature there. 

I'm open to be convinced otherwise, though. I doubt I have thought this through to the end ;)